examples/leaflet/large.html

```diff
@@ -138,7 +138,7 @@
         This is a map of every census block in the USA, including its
         population. The entire file is over 12GB, but FlatGeobuf fetches only
         the tiny subset of data that intersects with the bounding box (drawn
-        in yellow). Pan the map to move the query's bounding box.
+        in blue). Pan the map to move the query's bounding box.
```

Comment:
I changed the color of this a long time ago and forgot to update the corresponding text. 🤦